src/components/side_panel/conditional_formatting/cell_is_rule_editor.xml

```diff
@@ -73,37 +73,25 @@
           t-on-click="(ev) => this.toggleStyle('strikethrough', ev)">
           <t t-call="o-spreadsheet-Icon.STRIKE"/>
         </div>
-        <div class="o-tool o-dropdown o-with-color">
-          <span
-            title="TextColor"
-            t-attf-style="border-color:{{rule.style.textColor}}"
-            t-on-click.stop="(ev) => this.toggleMenu('cellIsRule-textColor', ev)">
-            <t t-call="o-spreadsheet-Icon.TEXT_COLOR"/>
-          </span>
-          <ColorPicker
-            t-if="state.openedMenu === 'cellIsRule-textColor'"
-            dropdownDirection="'center'"
-            onColorPicked="(color) => this.setColor('textColor', color)"
-            currentColor="rule.style.textColor"
-            t-key="textColor"
-          />
-        </div>
+        <ColorPickerEntry
+          currentColor="rule.style.textColor || '#000000'"
+          onClick="(ev) => this.toggleMenu('cellIsRule-textColor', ev)"
+          showColorPicker="state.openedMenu === 'cellIsRule-textColor'"
+          onColorPicked="(color) => this.setColor('textColor', color)"
+          tooltipText="'Text Color'"
```

Comment:
`tooltipText` should be translated I think. Pretty sure we didn't need to do it explicitely before because the `title` attribute is translated automatically.